### ISSUE TITLE
Move things out from the index store

### ISSUE THREAD

**Issue by hsjoberg** [hsjoberg/blixt-wallet#36]

**Comment by sesam** [comment]
Is this about IStoreModel ?

**Comment by hsjoberg** [comment]
@sesam Yes, right now it acts as both an root store covering all substores and also has it's own functionality.
Basically IStoreModel also handles: onboarding and common functionality, which could be moved to their own stores.
